rust/stackablectl/src/args/namespace.rs

```diff
@@ -0,0 +1,14 @@
+use clap::Args;
+use stackable_cockpit::constants::{DEFAULT_OPERATOR_NAMESPACE, DEFAULT_PRODUCT_NAMESPACE};
+
+#[derive(Debug, Args)]
+#[command(next_help_heading = "Namespace options")]
+pub struct CommonNamespaceArgs {
+    /// Namespace in the cluster used to deploy the operators
+    #[arg(short = 'n', long, default_value = DEFAULT_OPERATOR_NAMESPACE, global = true)]
```

Comment:
Please use the `short = 'n'` for the product namespaces

Reply:
Done in 7066caa